WorkflowAdminBundle/DependencyInjection/Configuration.php

```diff
@@ -32,11 +32,15 @@ public function getConfigTreeBuilder()
                 ->info('List of the color available, in the status for instance')
                 ->useAttributeAsKey('key')
                     ->defaultValue( array(
-                        'red'      => 'open_orchestra_workflow_admin.form.status.color.red',
-                        'green'    => 'open_orchestra_workflow_admin.form.status.color.green',
-                        'orange'   => 'open_orchestra_workflow_admin.form.status.color.orange',
-                        'grayDark' => 'open_orchestra_workflow_admin.form.status.color.grayDark',
-                        'blue'     => 'open_orchestra_workflow_admin.form.status.color.blue',
+                        'turquoise'   => 'open_orchestra_workflow_admin.form.status.color.turquoise',
+                        'emerald'     => 'open_orchestra_workflow_admin.form.status.color.emerald',
+                        'peter-river' => 'open_orchestra_workflow_admin.form.status.color.peter-river',
```

Comment:
the naming isn't very explicit for a developer, why not use the same than in translation